easybuild/easyblocks/generic/mesonninja.py

```diff
@@ -51,6 +53,9 @@ def extra_options(extra_vars=None):
         extra_vars.update({
             'configure_cmd': [DEFAULT_CONFIGURE_CMD, "Configure command to use", CUSTOM],
             'separate_build_dir': [True, "Perform build in a separate directory", CUSTOM],
+            'build_cmd': [DEFAULT_BUILD_CMD, "Build command to use", CUSTOM],
+            'build_dir': [None, "build_dir to pass to meson", CUSTOM],
+            'install_cmd': [DEFAULT_INSTALL_CMD, "Install command to use", CUSTOM],
```

Comment:
Let's retain alphabetical ordering here:

```suggestion
            'build_dir': [None, "build_dir to pass to meson", CUSTOM],
            'build_cmd': [DEFAULT_BUILD_CMD, "Build command to use", CUSTOM],
            'configure_cmd': [DEFAULT_CONFIGURE_CMD, "Configure command to use", CUSTOM],
            'install_cmd': [DEFAULT_INSTALL_CMD, "Install command to use", CUSTOM],
            'separate_build_dir': [True, "Perform build in a separate directory", CUSTOM],
```